src/components/skills-quiz/data/utils.jsx

```diff
@@ -13,6 +13,12 @@ export function sortSkillsWithSignificance(job) {
     (a.significance < b.significance) ? 1 : -1));
 }
 
+// returns skills based on # of courses in descending order
```

Comment:
Can we please make this comment a bit more descriptive and change this function name to `sortSkillsCoursesWithCourseCount `?

Reply:
Done